### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by jenting**
Signed-off-by: JenTing Hsiao <jenting.hsiao@suse.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,20 +27,25 @@ You can read more technical details of Longhorn [here](https://longhorn.io/).
...
-The latest release of Longhorn is **v1.1.0**.
+The latest release of Longhorn is **v1.1.1**.
 
 ## Source code
 Longhorn is 100% open source software. Project source code is spread across a number of repos:
 
-1. Longhorn engine -- Core controller/replica logic https://github.com/longhorn/longhorn-engine
-1. Longhorn manager -- Longhorn orchestration https://github.com/longhorn/longhorn-manager
-1. Longhorn UI -- Dashboard https://github.com/longhorn/longhorn-ui
+| Component                 | What it does                                                           | GitHub repo                                                                                 |
```
Comment (joshimoo) [inline README.md:42]: I like the addition of the table.

File: README.md [hunk 2]
```diff
@@ -27,20 +27,25 @@ You can read more technical details of Longhorn [here](https://longhorn.io/).
 
 ## Get Involved
 **Community Meeting and Office Hours**!: Hosted by the core maintainers of Longhorn: 2nd Friday of the every month at 09:00 Pacific Time (PT)/12:00 Eastern Time (ET) on Zoom: http://bit.ly/longhorn-community-meeting. Gcal event: http://bit.ly/longhorn-events
```
Comment (joshimoo) [inline README.md:29]: @innobead Are we switching back to zoom for the community meetings? If not we should update the reference as well.

File: README.md [hunk 3]
```diff
@@ -61,9 +66,10 @@ Longhorn can be installed on a Kubernetes cluster in several ways:
...
 
 ## Community
+
 Longhorn is open source software, so contributions are greatly welcome. Please read [Code of Conduct](./CODE_OF_CONDUCT.md) and [Contributing Guideline](./CONTRIBUTING.md) before contributing.
 
-Contributing code is not the only way of contributing. We value feedbacks very much and many of the Longhorn features are originated from users' feedback. If you have any feedbacks, feel free to [file an issue](https://github.com/longhorn/longhorn/issues/new?title=*Summarize%20your%20issue%20here*&body=*Describe%20your%20issue%20here*%0A%0A---%0AVersion%3A%20``) and talk to the developers at the [CNCF](https://slack.cncf.io/) [#longhorn](https://cloud-native.slack.com/messages/longhorn) slack channel.
+Contributing code is not the only way of contributing. We value feedbacks very much and many of the Longhorn features are originated from users' feedback. If you have any feedbacks, feel free to [file an issue](https://github.com/longhorn/longhorn/issues/new/choose) and talk to the developers at the [CNCF](https://slack.cncf.io/) [#longhorn](https://cloud-native.slack.com/messages/longhorn) slack channel.
```
Comment (joshimoo) [inline README.md:72]: Maybe add something about contributing to documentation, since the technical barrier is lower.

Comment (jenting) [reply README.md:72]: How about adding the https://github.com/longhorn/website under the repositories table?

File: README.md (hunk 2 above)
Comment (innobead) [reply README.md:29]: :) correct, we should fix it in another PR.

**Review by innobead — APPROVED** [review]

**Pull request merged**